server/boot/z-customer-accounts.js

```diff
@@ -0,0 +1,24 @@
+var async = require('async');
+
+module.exports = function(app) {
```

Comment:
hey @superkhau, I solved the async problem, now none of the urls rely on the creation sequence of any instances.